extension/src/lib/highContrast.js

```diff
@@ -0,0 +1,87 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+// This component makes use of css/highContrast.css to perform the adaptations
+// of the web content, and this is done through chrome.tabs.executeScript.
+//
+fluid.defaults("gpii.chrome.highContrast", {
+    gradeNames: ["fluid.modelComponent"],
+    scriptTemplate: "document.documentElement.setAttribute('data-gpii-hc','__THEME__');" +
+                    "[].forEach.call(document.querySelectorAll('body *'), function(node) {" +
+                        "node.setAttribute('data-gpii-hc', '__THEME__');" +
+                    "});" +
+                    "document.documentElement.setAttribute('data-gpii-hc', '__THEME__');",
+    disableScript: "document.documentElement.removeAttribute('data-gpii-hc');" +
+                    "[].forEach.call(document.querySelectorAll('body *'), function(node) {" +
+                        "node.removeAttribute('data-gpii-hc');" +
+                    "});",
+    commonToPlatform: {
+        highContrastTheme: {
+            transform: {
+                type: "fluid.transforms.valueMapper",
+                inputPath: "highContrastTheme",
+                defaultInputValue: "black-white",
+                options: {
+                    "black-white": {
+                        outputValue: "bw"
+                    },
+                    "white-black": {
+                        outputValue: "wb"
+                    },
+                    "black-yellow": {
+                        outputValue: "by"
+                    },
+                    "yellow-black": {
+                        outputValue: "yb"
+                    }
+                }
+            }
+        }
+    },
+    model: {
+        highContrastEnabled: undefined,
+        highContrastTheme: undefined
+    },
+    modelListeners: {
+        "*": {
+            funcName: "gpii.chrome.highContrast.modelChanged",
+            args: "{that}",
+            excludeSource: "init"
+        }
+    }
+});
+
+gpii.chrome.highContrast.modelChanged = function (that) {
+    if (that.model.highContrastEnabled) {
+        var theme = fluid.model.transformWithRules(that.model, that.options.commonToPlatform);
```

Comment:
Well, I've tried it but didn't work. :/ 
Is there any problem/restriction in using a valueMapper transformation inside a modelRelay?